CCFatigue_modules/2_S-NCurves/sn_curve_linlog.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python
+""" CCFatigue - Module 2 - sn_curve_linlog.py
+This code takes in AGG data and output SNC data
+"""
+
+import os
+import math
+import numpy as np
+import pandas as pd
+from scipy import stats
+from itertools import chain
+
+from astm import Astm
+
+
+CONFIDENCE = 95
+RELIABILITY_LEVEL = 50
+
+SRC_DIR = os.path.dirname(os.path.realpath(__file__))
+DATA_DIR = os.path.join(SRC_DIR, "..", "..", "Data")
+
+
+INPUT_FILENAME = "AGG_input.csv"
+INPUT_FILE = os.path.join(DATA_DIR, INPUT_FILENAME)
+OUTPUT_JSON_FILENAME = "SNC_LinLog.json"
+OUTPUT_JSON_FILE = os.path.join(DATA_DIR, OUTPUT_JSON_FILENAME)
+OUTPUT_CSV_FILENAME = "SNC_LinLog.csv"
+OUTPUT_CSV_FILE = os.path.join(DATA_DIR, OUTPUT_CSV_FILENAME)
+
+
+def equation_ycar(slope: float, intercept: float, stress_parameter: float) -> float:
+    """
+    https://github.com/EPFL-ENAC/CCFatiguePlatform/blob/develop/CCFatigue_modules/2_S-NCurves/S-N-Curve-LinLog.for#L314
+    Inputs:
+    - slope
+    - intercept
+    - stress_parameter
+    Outputs:
+    - ycar
+    """
+    ycar = slope + intercept * stress_parameter
+    return ycar
+
+
+def stress_at_failure(aa: float, bb: float, nn: float) -> float:
+    """
+    Calculate stress at failure
+    https://github.com/EPFL-ENAC/CCFatiguePlatform/blob/develop/CCFatigue_modules/2_S-NCurves/S-N-Curve-LinLog.for#L397
+    Inputs:
+    - aa
+    - bb
+    - nn
+    Outputs:
+    - spn: stress at failure
+    """

```

Comment:
Recently we found this standard syntax for Docstrings that works really well - especially for automating documentation later. Let's start using this ? Especially that ours is not super consistent between modules at the moment. 
 Can you also add the expected type of each params in the docstring?
```suggestion

 """
   Calculate stress at failure.

    Translated from : https://github.com/EPFL-ENAC/CCFatiguePlatform/blob/develop/CCFatigue_modules/2_S-NCurves/S-N-Curve-LinLog.for#L397

   Parameters
   ----------
   aa : float
       description of aa
   bb : float
       description of bb
   nn : float
       description of nn
   Returns
   -------
   spn : float
    stress at failure
   """

```

Reply:
Honne^tement les nommage de variables comme ça ("aa", "bb" ça ne va pas.. C'est pas du tout descriptif..Sont-elles nommées comme ça dans la référence?) Si on a compris ce que c'était, renommmer la vairbale un peu plus explicite permettrait de mieux lire ton code..

Reply:
spn par exemple, dans le variable_dictionary c'est sigma_max - pourquoi c'est spn ici ? c'est super dur à suivre.. Ca devrait être soit "sigma_max" (acronyme défini dans le variable_dictionary), soit "Stress_at_failure" (explicit variable naming